File: 브루트포스/프로그래머스Lv0-옹알이(1).js
function solution(babbling) {
  let total = 0;
  const possibleWords = ["aya", "ye", "woo", "ma"];

  babbling.forEach((word) => {
    possibleWords.forEach((possibleWord) => {
      word = word.replace(possibleWord, " ");
    });
    if (word.trim() === "") {
      total += 1;
    }
  });

  return total;
}
// 정확성  테스트
// 테스트 1 〉	통과 (0.18ms, 33.5MB)
// 테스트 2 〉	통과 (0.23ms, 33.5MB)
// 테스트 3 〉	통과 (0.21ms, 33.4MB)
// 테스트 4 〉	통과 (0.21ms, 33.5MB)
// 테스트 5 〉	통과 (0.23ms, 33.4MB)
// 테스트 6 〉	통과 (0.19ms, 33.5MB)
// 테스트 7 〉	통과 (0.19ms, 33.5MB)
// 테스트 8 〉	통과 (0.24ms, 33.5MB)
// 테스트 9 〉	통과 (0.18ms, 33.4MB)
// 테스트 10 〉	통과 (0.17ms, 33.6MB)
// 테스트 11 〉	통과 (0.16ms, 33.6MB)
// 테스트 12 〉	통과 (0.11ms, 33.5MB)

File: 브루트포스/프로그래머스Lv0-옹알이(1).py
def solution(babbling):
    total = 0
    possible_words = ["aya", "ye", "woo", "ma"]
    for word in babbling:
        for possible_word in possible_words:
            word = word.replace(possible_word, " ")
        if word.strip() == "":
            total += 1
    
    return total
"""
정확성  테스트
테스트 1 〉	통과 (0.02ms, 10.1MB)
테스트 2 〉	통과 (0.05ms, 10.1MB)
테스트 3 〉	통과 (0.04ms, 10.2MB)
테스트 4 〉	통과 (0.04ms, 10.1MB)
테스트 5 〉	통과 (0.04ms, 9.98MB)
테스트 6 〉	통과 (0.03ms, 10.3MB)
테스트 7 〉	통과 (0.03ms, 10.2MB)
테스트 8 〉	통과 (0.05ms, 9.97MB)
테스트 9 〉	통과 (0.03ms, 10.2MB)
테스트 10 〉	통과 (0.02ms, 10.2MB)
테스트 11 〉	통과 (0.01ms, 10.1MB)
테스트 12 〉	통과 (0.00ms, 10MB)
"""